src/test/java/gov/healthit/chpl/aqa/stepDefinitions/UserRegistrationSteps.java

```diff
@@ -0,0 +1,40 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.testng.Assert.assertEquals;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.UserRegistrationPage;
+/**
+ * Class UserRegistrationSteps definition.
+ */
+public class UserRegistrationSteps extends BaseSteps {
+    /**
+     * Get user to the User Registration Page.
+     * Click Create New Account
+     */
+    @Given("^I am on User Registration page$")
+    public void userRegistrationPage() {
+        WebDriver driver =  getDriver();
+        driver.get(getUrl() + "#/registration/create-user/laskdjfkdsaf");
+        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("window.scrollBy(0,500)");
```

Comment:
What's this scroll for? And why the value of 500? I'm always leery of seeing hard-coded numbers.

This is a "magic number", but checkstyle didn't catch it because it's inside a string